exec/cnex/lib/net.c

```diff
@@ -142,6 +142,7 @@ void net_socket_bind(TExecutor *exec)
     if (r < 0) {
         // ToDo: Raise exception here?
         perror("bind");
+        // Code will fall through to bail:, so a goto isn't necessary here.
```

Comment:
Don't take shortcuts, add the goto. That way: (1) the code will look right because it looks the same as all the other error cases, and (2) you won't get bit when someday you add some code after this if block.

Reply:
Fixed.